packages/protocol/scripts/utils.test.ts

```diff
@@ -0,0 +1,123 @@
+import { expect } from '@jest/globals'
+import { execSync } from 'child_process'
+import { SemVer } from 'semver'
+import { determineNextVersion, getReleaseTypeFromSemVer } from './utils'
+
+jest.mock('child_process', () => ({
+  execSync: jest.fn(),
+}))
+
+afterEach(() => {
+  jest.clearAllMocks()
+})
+
+const execSyncMock = execSync as jest.Mock
+
+const retrieveReleaseInformation = (version: SemVer): [string, string | number] => {
+  return [version.version, getReleaseTypeFromSemVer(version)]
+}
+
+describe('utils', () => {
+  describe('determineNextVersion()', () => {
+    it('determines post-audit release tag', () => {
```

Comment:
nit: i think for a reader this wording could be clarified as ' when given git tag with 'post-audit' determines the npm tag to be latest and gets the version from the git tag'

Reply:
provided more descriptive wording for both post- and pre-audit tags